data/processors/coords.py

```diff
@@ -2,6 +2,9 @@
 import logging
 
 import utm
+from utils import distance_via_great_circle
+
+MAX_DISTANCE_METERS_FROM_PARENT = 100
```

Comment:
Pushing this to `200` resolves two false positives

```suggestion
MAX_DISTANCE_METERS_FROM_PARENT = 200
```